build.sbt

```
@@ -1,8 +1,27 @@
+import ReleaseTransformations.*
+import sbtversionpolicy.withsbtrelease.ReleaseVersion
+
 organization := "com.gu"
 name := "dynamo-db-switches"
+licenses := Seq(License.Apache2)
 scalaVersion := "2.13.14"
+scalacOptions := Seq("-release:11")
```

Not sure if this makes any difference in practice.  Are there any default options set by SBT?
```suggestion
scalacOptions ++= Seq("-release:11")
```